django/db/migrations/writer.py

```diff
@@ -472,6 +490,8 @@ def serialize(cls, value):
                     "For more information, see "
                     "https://docs.djangoproject.com/en/%s/topics/migrations/#serializing-values"
                     % (value.__name__, module_name, get_docs_version()))
+            if module_name == '__builtin__':
```

Comment:
I must be missing something - `get_random_string` isn't a built-in function, so how could it be used to test this case?

I assumed since this code was in here, you had a real-world use case for needing to serialize a built-in function in a migration. Could that use case be turned into a test?

If not, maybe this isn't needed?